setup.py

```diff
@@ -59,8 +59,9 @@
     packages=find_packages(),
     include_dirs=[np.get_include()],
     package_data={
-        'retro.tables': '*.pyx'
```

Comment:
My mistake: I was thinking "pyc" when I said you shouldn't include "pyx", but we _do_ want "pyx" files (these are cython files)